### ISSUE TITLE
QUAST cannot be installed

### ISSUE THREAD

**Issue by iquasere** [bioconda/bioconda-recipes#23424]
I talked about this in the gitter, but think it should be made an issue, as it is seemingly stopping use of QUAST through conda. Running ```conda install -y -c bioconda quast``` gives me
```
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /home/jsequeira/anaconda3

  added / updated specs:
    - quast


The following NEW packages will be INSTALLED:

  circos             bioconda/noarch::circos-0.69.8-0
  giflib             conda-forge/linux-64::giflib-5.1.7-h516909a_1
  glimmerhmm         bioconda/linux-64::glimmerhmm-3.0.4-pl526hf484d3e_3
  k8                 bioconda/linux-64::k8-0.2.5-he513fc3_0
  libgd              pkgs/main/linux-64::libgd-2.2.5-h0db58fd_3
  libwebp            pkgs/main/linux-64::libwebp-1.0.1-h8e7db2f_0
  minimap2           bioconda/linux-64::minimap2-2.17-hed695b0_3
  perl-autoloader    bioconda/linux-64::perl-autoloader-5.74-pl526_2
  perl-clone         bioconda/linux-64::perl-clone-0.42-pl526h516909a_0
  perl-config-gener~ bioconda/linux-64::perl-config-general-2.63-pl526_0
  perl-digest-perl-~ bioconda/linux-64::perl-digest-perl-md5-1.9-pl526_1
  perl-dynaloader    bioconda/linux-64::perl-dynaloader-1.25-pl526_1
  perl-font-ttf      bioconda/linux-64::perl-font-ttf-1.06-pl526_0
  perl-gd            bioconda/linux-64::perl-gd-2.71-pl526he860b03_0
  perl-io-string     bioconda/linux-64::perl-io-string-1.08-pl526_3
  perl-math-bezier   bioconda/linux-64::perl-math-bezier-0.01-pl526_1
  perl-math-round    bioconda/linux-64::perl-math-round-0.07-pl526_1
  perl-math-vecstat  bioconda/linux-64::perl-math-vecstat-0.08-pl526_1
  perl-module-imple~ bioconda/linux-64::perl-module-implementation-0.09-pl526_2
  perl-module-runti~ bioconda/linux-64::perl-module-runtime-0.016-pl526_1
  perl-number-format bioconda/linux-64::perl-number-format-1.75-pl526_3
  perl-params-valid~ bioconda/linux-64::perl-params-validate-1.29-pl526h14c3975_1
  perl-readonly      bioconda/linux-64::perl-readonly-2.05-pl526_0
  perl-regexp-common bioconda/linux-64::perl-regexp-common-2017060201-pl526_0
  perl-set-intspan   bioconda/linux-64::perl-set-intspan-1.19-pl526_1
  perl-statistics-b~ bioconda/linux-64::perl-statistics-basic-1.6611-pl526_2
  perl-svg           bioconda/linux-64::perl-svg-2.84-pl526_0
  perl-text-format   bioconda/linux-64::perl-text-format-0.59-pl526_2
  perl-time-hires    bioconda/linux-64::perl-time-hires-1.9760-pl526h14c3975_1
  quast              bioconda/linux-64::quast-5.0.2-py37pl526hb5aa323_2
  simplejson         conda-forge/linux-64::simplejson-3.17.2-py37h8f50634_0


Preparing transaction: done
Verifying transaction: done
Executing transaction: | b'The default QUAST package does not include:\n* GRIDSS (needed for structural variants detection)\n* SILVA 16S rRNA database (needed for reference genome detection in metagenomic datasets)\n* BUSCO tools and databases (needed for searching BUSCO genes) -- works in Linux only!\n\nTo be able to use those, please run\n    quast-download-gridss\n    quast-download-silva\n    quast-download-busco\n'
done
ERROR conda.core.link:_execute(700): An error occurred while installing package 'bioconda::perl-digest-perl-md5-1.9-pl526_1'.
Rolling back transaction: done

[Errno 13] Permission denied: '/home/jsequeira/anaconda3/lib/site_perl/5.26.2/Digest/Perl'
()
```
Looking at the QUAST [recipe page ](https://bioconda.github.io/recipes/quast/README.html) shows that QUAST is only available for python < 2.8. However, in its recipe I don't see that specified. Why is it limited to python < 2.8? And how can I make it available for python 3?

**Comment by iquasere** [comment]
Just a lack of permissions on the Conda install folder. Fixed with ```sudo chown -R user ~/anaconda3```

**Comment by mbargull** [comment]
> Looking at the QUAST [recipe page ](https://bioconda.github.io/recipes/quast/README.html) shows that QUAST is only available for python < 2.8. However, in its recipe I don't see that specified. Why is it limited to python < 2.8? And how can I make it available for python 3?

Apparently, this picked the constraints from one of the `quast` builds, which happened to be the `python=2.7` one. Ideally, we should list multiple versions for the dependencies that are required by all of the latest builds.
Could you open an issue over at https://github.com/bioconda/bioconda-utils (the docs as well as package/recipe list is generated from that repository)?

**Comment by iquasere** [comment]
Raised an [issue ](https://github.com/bioconda/bioconda-utils/issues/669) after you suggested it, forgot to note it here!
